src/pages/Landing.tsx

```diff
@@ -14,25 +14,18 @@ function Landing() {
             <img src={BackgroundTopDetail} alt="background top detail" className="background-top-detail"/>
             <img src={BackgroundBottomDetail} alt="background bottom detail" className="background-bottom-detail"/>
             <div className="content-wrapper">
-                <main>
-                    <div className="home_content">
-                        <div className="grid_column">
-                            <div className="main-page-logo">
-                                <SiteLogo className="" width={277} height={166} />
-                            </div>
-                            <div>
-                                <div className="text_grid">
-                                    <h1>Our lab to try tech</h1>
-                                    <div className="green-dot"></div>
-                                </div>
-                            </div>
-                            <p>We're a group of friends trying to do cool things.</p>
-                        </div>
-                        <div className="grid_column">
-                            <img src={Developer} alt="Man in front of a laptop" className="home-developer" />
-                        </div>
+                <div className="main-page-logo">
+                    <SiteLogo className="" width={277} height={166} />
```

Comment:
The only change I'd recomend is to move width and height for the css file creating a new className or just remove the className

Reply:
I could @mspdl, but the idea on that is to reuse the component. If you take a look on `Landing.tsx` and `NavigationMenu.tsx` you'll see that I'm using this SiteLogo component on both, changing only it's size (`width` and `height`). That's why I've created it outside a css class. 🙂